nni/experiment/experiment.py

```diff
@@ -87,20 +87,10 @@ def __init__(self, config_or_platform: ExperimentConfig | str | list[str] | None
         else:
             self.config = config_or_platform
 
-    def start(self, port: int = 8080, debug: bool = False, run_mode: RunMode = RunMode.Background) -> None:
-        """
-        Start the experiment in background.
-
-        This method will raise exception on failure.
-        If it returns, the experiment should have been successfully started.
-
-        Parameters
-        ----------
-        port
-            The port of web UI.
-        debug
-            Whether to start in debug mode.
-        """
+    def _start_impl(self, port: int, debug: bool, run_mode: RunMode,
+                    url_prefix: str | None,
```

Comment:
I think `url_prefix` is a field `self.url_prefix`.

Reply:
do we have unit test for url_prefix?

Reply:
in the previous version of retiarii experiment, url_prefix is None? BTW, how to configure this field?

Reply:
You can set `self.url_prefix` to `None`. This is an undocumented feature in Python API, but is documented in `nnictl`.
In NNI manager it does have unit test. But I cannot remember whether it has integration test. Since not documented in Python API it should be tested with `nnictl create`, and the first version was not implemented by me.